medor_custom/__openerp__.py

```diff
@@ -15,6 +15,7 @@
     "category": "Website",
     "version": "9.0.1.0",
     "depends": [
+        "l10n_be",
```

Comment:
so you made it for belgium only ?

Reply:
yes, c'est le module `medor_custom`, c'est que pour medor qui est en belgique